Baikal/Controllers/clw_scene_controller.cpp

```diff
@@ -227,7 +227,7 @@ namespace Baikal
             auto transform = mesh->GetTransform();
             shape->SetTransform(transform, inverse(transform));
             shape->SetId(id++);
-            shape->SetMask(iter->GetVisibilityMask());
+            // shape->SetMask(iter->GetVisibilityMask());
```

Comment:
Are you sure we don't need this?

Reply:
CI fails to build with this line uncommented.